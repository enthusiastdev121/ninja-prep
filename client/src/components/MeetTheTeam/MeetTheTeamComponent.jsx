import React from 'react'
import { MeetTheTeamDesktop } from './MeetTheTeamDesktop'
import { MeetTheTeamTablet } from './MeetTheTeamTablet'
import { MeetTheTeamMobile } from './MeetTheTeamMobile'
import { Desktop, Tablet, Mobile } from '../MediaQuery/DeviceSizes'

const MeetTheTeamComponent = () => (
    <div>
        <Desktop>
            <MeetTheTeamDesktop />
        </Desktop>
        <Tablet>
            <MeetTheTeamTablet />
        </Tablet>
        <Mobile>
            <MeetTheTeamMobile />
        </Mobile>
    </div>
)

export default MeetTheTeamComponent
export { default as MeetTheTeamComponent } from './MeetTheTeamComponent'
