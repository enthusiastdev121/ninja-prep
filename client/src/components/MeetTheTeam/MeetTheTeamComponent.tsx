import React from 'react';

import {Desktop, Mobile} from '../../utils/DeviceSizes';
import MeetTheTeamDesktop from './MeetTheTeamDesktop';
import MeetTheTeamMobile from './MeetTheTeamMobile';

const MeetTheTeamComponent = (): JSX.Element => (
  <div>
    <Desktop>
      <MeetTheTeamDesktop />
    </Desktop>
    <Mobile>
      <MeetTheTeamMobile />
    </Mobile>
  </div>
);

export default MeetTheTeamComponent;
