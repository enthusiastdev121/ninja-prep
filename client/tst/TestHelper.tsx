import React from 'react'
import { Router } from 'react-router-dom'
import { createBrowserHistory } from 'history'
const history = createBrowserHistory()

interface DEVICE_WIDTHS {
    readonly DESKTOP: number
    readonly MOBILE: number
    readonly TABLET: number
}

//Widths in Pixels. Aligns with Media Query widths
export const DEVICE_TYPES: DEVICE_WIDTHS = {
    DESKTOP: 1000,
    MOBILE: 750,
    TABLET: 800
}

export const routeWrapper = (children: React.ReactChild) => {
    return <Router history={history}>{children}</Router>
}
