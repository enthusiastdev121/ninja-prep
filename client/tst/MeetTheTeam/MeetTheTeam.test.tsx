import React from 'react'
import { shallow, mount } from 'enzyme'
import { Context as ResponsiveContext } from 'react-responsive'
import { DEVICE_TYPES } from '../TestHelper'
import toJson from 'enzyme-to-json'
import MeetTheTeamComponent from '../../src/components/MeetTheTeam/MeetTheTeamComponent'

test.each(Object.entries(DEVICE_TYPES))('renders successfully on %p', (deviceType: string, deviceWidth: number) => {
    const wrappingComponent = ResponsiveContext.Provider
    const wrappingComponentProps = { value: { width: deviceWidth } }
    const mountProps = { wrappingComponent, wrappingComponentProps }

    const wrapper = shallow(<MeetTheTeamComponent />, mountProps)
    expect(wrapper).toBeDefined
})

test.each(Object.entries(DEVICE_TYPES))('matches snapshot on %p', (deviceType: string, deviceWidth: number) => {
    const wrappingComponent = ResponsiveContext.Provider
    const wrappingComponentProps = { value: { width: deviceWidth } }
    const mountProps = { wrappingComponent, wrappingComponentProps }

    const wrapper = mount(<MeetTheTeamComponent />, mountProps)
    expect(toJson(wrapper)).toMatchSnapshot()
})
