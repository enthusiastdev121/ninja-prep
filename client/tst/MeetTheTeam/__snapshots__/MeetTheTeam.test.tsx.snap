// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot on "DESKTOP" 1`] = `
<MeetTheTeamPage>
  <Responsiveness
    desktop={<MeetTheTeamDesktop />}
    mobile={<MeetTheTeamMobile />}
  >
    <Desktop>
      <MeetTheTeamDesktop>
        <div>
          <MeetTheTeamHeaderDesktop>
            <styled.div>
              <div
                className="sc-bdnxRM ejnGdQ py-5"
              >
                <DarkNinjaPrepLogo
                  height={150}
                  width={500}
                >
                  <svg
                    height={150}
                    viewBox="0 0 201 58"
                    width={500}
                  >
                    <g
                      data-name="Group 119"
                      id="Group_119"
                      transform="translate(-1157 -10129)"
                    >
                      <text
                        fill="#2a2a3a"
                        fontFamily="SourceSansPro-Bold, Source Sans Pro"
                        fontSize="41"
                        fontWeight="700"
                        id="NinjaPrep"
                        transform="translate(1157 10171)"
                      >
                        <tspan
                          x="0"
                          y="0"
                        >
                          NinjaPrep
                        </tspan>
                      </text>
                      <circle
                        cx="7.5"
                        cy="7.5"
                        data-name="Ellipse 44"
                        fill="#25d0f3"
                        id="Ellipse_44"
                        r="7.5"
                        transform="translate(1343 10158)"
                      />
                    </g>
                  </svg>
                </DarkNinjaPrepLogo>
                <styled.h1>
                  <h1
                    className="sc-dlnjwi display-5 font-weight-bold pb-2"
                  >
                    Meet The Team
                  </h1>
                </styled.h1>
                <styled.p>
                  <p
                    className="sc-gtsrHT crKwMa text-secondary lead"
                  >
                    A place to help you build a strong foundation in data structures and
                  </p>
                </styled.p>
                <styled.p>
                  <p
                    className="sc-gtsrHT crKwMa text-secondary lead"
                  >
                    algorithms and prepare for technical interviews, all in one platform.
                  </p>
                </styled.p>
              </div>
            </styled.div>
          </MeetTheTeamHeaderDesktop>
          <MeetTheTeamInfoDesktop>
            <Styled(Col)>
              <Col
                className="sc-hKFxyN ecGXkA"
              >
                <div
                  className="sc-hKFxyN ecGXkA col"
                >
                  <Styled(Row)>
                    <Row
                      className="sc-eCApnc hDmciy"
                      noGutters={false}
                    >
                      <div
                        className="sc-eCApnc hDmciy row"
                      >
                        <Styled(Col)
                          xs={8}
                        >
                          <Col
                            className="sc-jSFjdj iJtmkD"
                            xs={8}
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col-8"
                            >
                              <styled.h1>
                                <h1
                                  className="sc-gKAaRy display-4 pb-3 font-weight-bold"
                                >
                                  Alex Nguyen
                                </h1>
                              </styled.h1>
                              <styled.h1>
                                <h1
                                  className="sc-iCoGMd cLnfWA lead text-secondary"
                                >
                                  Alex Nguyen is the founder and content creator of NinjaPrep. He is currently a software engineer who has previously received offers from both Amazon and Microsoft. Alex graduated from New York University in 2019 with a degree in Computer Science with a focus on full-stack development. After endless practice preparing for algorithm questions, he wanted to provide an opportunity for all audiences to learn algorithmic problem solving as simply as possible. When he's not coding, he does photography and enjoys spending time with the friends around him.
                                </h1>
                              </styled.h1>
                            </div>
                          </Col>
                        </Styled(Col)>
                        <Styled(Col)>
                          <Col
                            className="sc-jSFjdj iJtmkD"
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col"
                            >
                              <img
                                alt="AlexPhoto"
                                className="img-fluid rounded"
                                src="/images/ALEX-PHOTO.jpg"
                              />
                            </div>
                          </Col>
                        </Styled(Col)>
                      </div>
                    </Row>
                  </Styled(Row)>
                  <Styled(Row)>
                    <Row
                      className="sc-eCApnc hDmciy"
                      noGutters={false}
                    >
                      <div
                        className="sc-eCApnc hDmciy row"
                      >
                        <Styled(Col)>
                          <Col
                            className="sc-jSFjdj iJtmkD"
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col"
                            >
                              <img
                                alt="AngelaSPhoto"
                                className="img-fluid rounded"
                                src="/images/ANGELA-S-PHOTO.jpg"
                              />
                            </div>
                          </Col>
                        </Styled(Col)>
                        <Styled(Col)
                          xs={8}
                        >
                          <Col
                            className="sc-jSFjdj iJtmkD"
                            xs={8}
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col-8"
                            >
                              <styled.h1>
                                <h1
                                  className="sc-gKAaRy display-4 pb-3 font-weight-bold"
                                >
                                  Angela Sun
                                </h1>
                              </styled.h1>
                              <styled.h1>
                                <h1
                                  className="sc-iCoGMd cLnfWA lead text-secondary"
                                >
                                  Angela Sun is a recent Computer Science Bachelor's graduate of the University of Washington in Seattle and also studied a semester at KTH in Sweden. She currently works as a software engineer at Bloomberg L.P. She has been contributing to developing NinjaPrep's user interface and content. Outside of work, she enjoys cooking, painting, and dancing.
                                </h1>
                              </styled.h1>
                            </div>
                          </Col>
                        </Styled(Col)>
                      </div>
                    </Row>
                  </Styled(Row)>
                  <Styled(Row)>
                    <Row
                      className="sc-eCApnc hDmciy"
                      noGutters={false}
                    >
                      <div
                        className="sc-eCApnc hDmciy row"
                      >
                        <Styled(Col)
                          xs={8}
                        >
                          <Col
                            className="sc-jSFjdj iJtmkD"
                            xs={8}
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col-8"
                            >
                              <styled.h1>
                                <h1
                                  className="sc-gKAaRy display-4 pb-3 font-weight-bold"
                                >
                                  Angela Yeung
                                </h1>
                              </styled.h1>
                              <styled.h1>
                                <h1
                                  className="sc-iCoGMd cLnfWA lead text-secondary"
                                >
                                  Angela Yeung is currently a student at New York University majoring in Computer Science earning her Bachelor's Degree. As a student herself, Angela uses the provided resources to practice algorithm problems and is passionate about creating a way for others to learn more about coding. She is a content contributor at NinjaPrep and has been developing NinjaPrep's back-end server as well. In her free time, Angela enjoys listening to music and finding new places to eat at.
                                </h1>
                              </styled.h1>
                            </div>
                          </Col>
                        </Styled(Col)>
                        <Styled(Col)>
                          <Col
                            className="sc-jSFjdj iJtmkD"
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col"
                            >
                              <img
                                alt="AngelaYPhoto"
                                className="img-fluid rounded"
                                src="/images/ANGELA-Y-PHOTO.jpg"
                              />
                            </div>
                          </Col>
                        </Styled(Col)>
                      </div>
                    </Row>
                  </Styled(Row)>
                  <Styled(Row)>
                    <Row
                      className="sc-eCApnc hDmciy"
                      noGutters={false}
                    >
                      <div
                        className="sc-eCApnc hDmciy row"
                      >
                        <Styled(Col)>
                          <Col
                            className="sc-jSFjdj iJtmkD"
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col"
                            >
                              <img
                                alt="JoannePhoto"
                                className="img-fluid rounded"
                                src="/images/JOANNE-PHOTO.jpg"
                              />
                            </div>
                          </Col>
                        </Styled(Col)>
                        <Styled(Col)
                          xs={8}
                        >
                          <Col
                            className="sc-jSFjdj iJtmkD"
                            xs={8}
                          >
                            <div
                              className="sc-jSFjdj iJtmkD col-8"
                            >
                              <styled.h1>
                                <h1
                                  className="sc-gKAaRy display-4 pb-3 font-weight-bold"
                                >
                                  Joanne Cheong
                                </h1>
                              </styled.h1>
                              <styled.h1>
                                <h1
                                  className="sc-iCoGMd cLnfWA lead text-secondary"
                                >
                                  Joanne Cheong pursued her Bachelor's in Management Information Systems at the University of Connecticut. She enjoyed working at the school's innovation lab, where she taught students data analytics and curated online courses. Since her graduation in December 2019, she has been working with Pfizer as part of their Digital Rotational program, focusing on project management, automation and user experience. She has decided to learn full-stack development and has been applying her experiences to develop NinjaPrep's user interface and business operations.
                                </h1>
                              </styled.h1>
                            </div>
                          </Col>
                        </Styled(Col)>
                      </div>
                    </Row>
                  </Styled(Row)>
                </div>
              </Col>
            </Styled(Col)>
          </MeetTheTeamInfoDesktop>
        </div>
      </MeetTheTeamDesktop>
    </Desktop>
    <Mobile />
  </Responsiveness>
</MeetTheTeamPage>
`;

exports[`matches snapshot on "MOBILE" 1`] = `
<MeetTheTeamPage>
  <Responsiveness
    desktop={<MeetTheTeamDesktop />}
    mobile={<MeetTheTeamMobile />}
  >
    <Desktop />
    <Mobile>
      <MeetTheTeamMobile>
        <div>
          <MeetTheTeamHeaderMobile>
            <Container
              fluid={false}
            >
              <div
                className="container"
              >
                <styled.div>
                  <div
                    className="sc-fujyAs isvnWl"
                  >
                    <DarkNinjaPrepLogo
                      height={175}
                      width={325}
                    >
                      <svg
                        height={175}
                        viewBox="0 0 201 58"
                        width={325}
                      >
                        <g
                          data-name="Group 119"
                          id="Group_119"
                          transform="translate(-1157 -10129)"
                        >
                          <text
                            fill="#2a2a3a"
                            fontFamily="SourceSansPro-Bold, Source Sans Pro"
                            fontSize="41"
                            fontWeight="700"
                            id="NinjaPrep"
                            transform="translate(1157 10171)"
                          >
                            <tspan
                              x="0"
                              y="0"
                            >
                              NinjaPrep
                            </tspan>
                          </text>
                          <circle
                            cx="7.5"
                            cy="7.5"
                            data-name="Ellipse 44"
                            fill="#25d0f3"
                            id="Ellipse_44"
                            r="7.5"
                            transform="translate(1343 10158)"
                          />
                        </g>
                      </svg>
                    </DarkNinjaPrepLogo>
                    <styled.h1>
                      <h1
                        className="sc-pNWdM display-4 font-weight-bold pb-2 pt-0 mt-0"
                      >
                        Meet The Team
                      </h1>
                    </styled.h1>
                    <styled.p>
                      <p
                        className="sc-jrsJWt text-secondary lead"
                      >
                        A place to help you build a strong foundation in data structures and algorithms and prepare for technical interviews, all in one platform.
                      </p>
                    </styled.p>
                  </div>
                </styled.div>
              </div>
            </Container>
          </MeetTheTeamHeaderMobile>
          <MeetTheTeamInfoMobile>
            <Styled(Container)>
              <Container
                className="sc-crzoAE py-3 w-75"
                fluid={false}
              >
                <div
                  className="sc-crzoAE py-3 w-75 container"
                >
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Alex Nguyen
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AlexPhoto"
                          className="img-fluid rounded"
                          src="/images/ALEX-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Alex Nguyen is the founder and content creator of NinjaPrep. He is currently a software engineer who has previously received offers from both Amazon and Microsoft. Alex graduated from New York University in 2019 with a degree in Computer Science with a focus on full-stack development. After endless practice preparing for algorithm questions, he wanted to provide an opportunity for all audiences to learn algorithmic problem solving as simply as possible. When he's not coding, he does photography and enjoys spending time with the friends around him.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Angela Sun
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AngelaSPhoto"
                          className="img-fluid rounded"
                          src="/images/ANGELA-S-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Angela Sun is a recent Computer Science Bachelor's graduate of the University of Washington in Seattle and also studied a semester at KTH in Sweden. She currently works as a software engineer at Bloomberg L.P. She has been contributing to developing NinjaPrep's user interface and content. Outside of work, she enjoys cooking, painting, and dancing.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Angela Yeung
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AngelaYPhoto"
                          className="img-fluid rounded"
                          src="/images/ANGELA-Y-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Angela Yeung is currently a student at New York University majoring in Computer Science earning her Bachelor's Degree. As a student herself, Angela uses the provided resources to practice algorithm problems and is passionate about creating a way for others to learn more about coding. She is a content contributor at NinjaPrep and has been developing NinjaPrep's back-end server as well. In her free time, Angela enjoys listening to music and finding new places to eat at.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Joanne Cheong
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="JoannePhoto"
                          className="img-fluid rounded"
                          src="/images/JOANNE-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Joanne Cheong pursued her Bachelor's in Management Information Systems at the University of Connecticut. She enjoyed working at the school's innovation lab, where she taught students data analytics and curated online courses. Since her graduation in December 2019, she has been working with Pfizer as part of their Digital Rotational program, focusing on project management, automation and user experience. She has decided to learn full-stack development and has been applying her experiences to develop NinjaPrep's user interface and business operations.
                      </p>
                    </styled.p>
                  </div>
                </div>
              </Container>
            </Styled(Container)>
          </MeetTheTeamInfoMobile>
        </div>
      </MeetTheTeamMobile>
    </Mobile>
  </Responsiveness>
</MeetTheTeamPage>
`;

exports[`matches snapshot on "TABLET" 1`] = `
<MeetTheTeamPage>
  <Responsiveness
    desktop={<MeetTheTeamDesktop />}
    mobile={<MeetTheTeamMobile />}
  >
    <Desktop />
    <Mobile>
      <MeetTheTeamMobile>
        <div>
          <MeetTheTeamHeaderMobile>
            <Container
              fluid={false}
            >
              <div
                className="container"
              >
                <styled.div>
                  <div
                    className="sc-fujyAs isvnWl"
                  >
                    <DarkNinjaPrepLogo
                      height={175}
                      width={325}
                    >
                      <svg
                        height={175}
                        viewBox="0 0 201 58"
                        width={325}
                      >
                        <g
                          data-name="Group 119"
                          id="Group_119"
                          transform="translate(-1157 -10129)"
                        >
                          <text
                            fill="#2a2a3a"
                            fontFamily="SourceSansPro-Bold, Source Sans Pro"
                            fontSize="41"
                            fontWeight="700"
                            id="NinjaPrep"
                            transform="translate(1157 10171)"
                          >
                            <tspan
                              x="0"
                              y="0"
                            >
                              NinjaPrep
                            </tspan>
                          </text>
                          <circle
                            cx="7.5"
                            cy="7.5"
                            data-name="Ellipse 44"
                            fill="#25d0f3"
                            id="Ellipse_44"
                            r="7.5"
                            transform="translate(1343 10158)"
                          />
                        </g>
                      </svg>
                    </DarkNinjaPrepLogo>
                    <styled.h1>
                      <h1
                        className="sc-pNWdM display-4 font-weight-bold pb-2 pt-0 mt-0"
                      >
                        Meet The Team
                      </h1>
                    </styled.h1>
                    <styled.p>
                      <p
                        className="sc-jrsJWt text-secondary lead"
                      >
                        A place to help you build a strong foundation in data structures and algorithms and prepare for technical interviews, all in one platform.
                      </p>
                    </styled.p>
                  </div>
                </styled.div>
              </div>
            </Container>
          </MeetTheTeamHeaderMobile>
          <MeetTheTeamInfoMobile>
            <Styled(Container)>
              <Container
                className="sc-crzoAE py-3 w-75"
                fluid={false}
              >
                <div
                  className="sc-crzoAE py-3 w-75 container"
                >
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Alex Nguyen
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AlexPhoto"
                          className="img-fluid rounded"
                          src="/images/ALEX-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Alex Nguyen is the founder and content creator of NinjaPrep. He is currently a software engineer who has previously received offers from both Amazon and Microsoft. Alex graduated from New York University in 2019 with a degree in Computer Science with a focus on full-stack development. After endless practice preparing for algorithm questions, he wanted to provide an opportunity for all audiences to learn algorithmic problem solving as simply as possible. When he's not coding, he does photography and enjoys spending time with the friends around him.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Angela Sun
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AngelaSPhoto"
                          className="img-fluid rounded"
                          src="/images/ANGELA-S-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Angela Sun is a recent Computer Science Bachelor's graduate of the University of Washington in Seattle and also studied a semester at KTH in Sweden. She currently works as a software engineer at Bloomberg L.P. She has been contributing to developing NinjaPrep's user interface and content. Outside of work, she enjoys cooking, painting, and dancing.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Angela Yeung
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="AngelaYPhoto"
                          className="img-fluid rounded"
                          src="/images/ANGELA-Y-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Angela Yeung is currently a student at New York University majoring in Computer Science earning her Bachelor's Degree. As a student herself, Angela uses the provided resources to practice algorithm problems and is passionate about creating a way for others to learn more about coding. She is a content contributor at NinjaPrep and has been developing NinjaPrep's back-end server as well. In her free time, Angela enjoys listening to music and finding new places to eat at.
                      </p>
                    </styled.p>
                  </div>
                  <div>
                    <styled.h1>
                      <h1
                        className="sc-iqAclL display-5  text-center"
                      >
                        Joanne Cheong
                      </h1>
                    </styled.h1>
                    <styled.div>
                      <div
                        className="sc-dIsUp huUEVJ"
                      >
                        <img
                          alt="JoannePhoto"
                          className="img-fluid rounded"
                          src="/images/JOANNE-PHOTO.jpg"
                        />
                      </div>
                    </styled.div>
                    <styled.p>
                      <p
                        className="sc-kEqXSa lead my-5 text-secondary"
                      >
                        Joanne Cheong pursued her Bachelor's in Management Information Systems at the University of Connecticut. She enjoyed working at the school's innovation lab, where she taught students data analytics and curated online courses. Since her graduation in December 2019, she has been working with Pfizer as part of their Digital Rotational program, focusing on project management, automation and user experience. She has decided to learn full-stack development and has been applying her experiences to develop NinjaPrep's user interface and business operations.
                      </p>
                    </styled.p>
                  </div>
                </div>
              </Container>
            </Styled(Container)>
          </MeetTheTeamInfoMobile>
        </div>
      </MeetTheTeamMobile>
    </Mobile>
  </Responsiveness>
</MeetTheTeamPage>
`;
